cypress/helpers/table.js

```diff
@@ -1,5 +1,12 @@
 import { EXTENDED_TIMEOUT } from '../support/util.js'
 
+export const expectAOTitleToBeValue = (value) =>
```

Comment:
Since it is checking "contain", maybe the function title should be `expectAOTitleToContain`